db/organizations/bilibili.eno

```diff
@@ -0,0 +1,7 @@
+name: Bilibili
+website_url: https://www.bilibili.com
+privacy_policy_url: https://www.bilibili.com/blackboard/privacy-pc.html
+privacy_contact: help@bilibili.com
+country: CN
+description: Video streaming and sharing platform
```

Comment:
Since it has a Wikipedia entry, we could use a shortened version of it: https://en.wikipedia.org/wiki/Bilibili

For instance:

```
Bilibili, nicknamed B Site, is a video-sharing website based in Shanghai where users can submit, view, and add overlaid commentary on videos.
```